paystell-frontend/src/hooks/use-wallet.tsx

```
@@ -0,0 +1,255 @@
+        payments: action.payload.records,
+        paymentCursor: action.payload.next,
+        isLoadingPayments: false,
+      }
+    case "FETCH_PAYMENTS_FAILURE":
+      return { ...state, paymentError: action.payload, isLoadingPayments: false }
+    case "SET_PAYMENT_CURSOR":
+      return { ...state, paymentCursor: action.payload }
+    case "ADD_PREV_CURSOR":
+      return { ...state, prevCursors: [...state.prevCursors, action.payload] }
+    case "POP_PREV_CURSOR":
+      const newPrevCursors = [...state.prevCursors]
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)